procedures/unit-testing-basics.ipf

```diff
@@ -1182,7 +1147,7 @@ static function GetTestCaseCount(testCaseList, procWin)
 		if(UTF_FuncRefIsAssigned(FuncRefInfo(TestCaseFunc)))
 			tcCount += 1
 		else
-			dgenFuncName = GetFunctionTag(testCase, UTF_TD_GENERATOR, UTF_TD_GENERATOR_L)
+			dgenFuncName = UTF_Utils#GetFunctionTagValue(testCase, UTF_FTAG_TD_GENERATOR, var)
```

Comment:
The problem is only if a data generator cannot be found. However, at the beginning of RunTest, when GetTestCasesMatch is called for the first time, all function tags are read and checked if all data generators can be found, otherwise it will abort (when issue#99 will be done). So in my eyes there is no way UTF_Utils#GetFunctionTagValue returns an error at these positions.